candi.sh

```diff
@@ -54,7 +54,7 @@ while [ -n "$1" ]; do
 	    echo "Usage: $0 [options]"
 	    echo "Options:"
 	    echo "  -p <path>, --prefix=<path>  set a different prefix path (default $PREFIX)"
-	    echo "  -j <N>, -j<N>, --PROCS=<N>  compile with N processes in parallel (default $PROCS)"
+	    echo "  -j <N>, -j<N>, --jobs=<N>   compile with N processes in parallel (default ${PROCS})"
```

Comment:
Change request: "compile with up to N jobs in parallel (default ${JOBS})"

Reply:
PROCS is a variable defined in line 43. Or do you mean to replace also this variable name completely?

Reply:
yes, renaming it consistently would make sense

Reply:
Now it consistently uses JOBS instead of PROCS as variable.